### PR TITLE
Feat/sqs

### PR DESCRIPTION
**Pull request by leonprou**
## Proposed changes
- adding SQS queue support
- introducing V3 API that uses SQS

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: server/config/custom-environment-variables.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
       "region": "COMMUNITY_AWS_SNS_REGION",
       "senderId": "COMMUNITY_AWS_SNS_SENDER_ID",
       "smsType": "COMMUNITY_AWS_SNS_SMS_TYPE"
+    },
+    "sqs": {
+      "queueUrl": "COMMUNITY_AWS_SQS_QUEUE_URL"
```
Comment (leonprou) [inline server/config/custom-environment-variables.json:40]: need to add to ECS

File: server/config/default.js [hunk 2]
```diff
@@ -251,6 +251,15 @@ module.exports = {
     },
     s3: {
       bucket: 'fuse-studio-qa'
+    },
+    sqs: {
+      constructor: {
```
Comment (leonprou) [inline server/config/default.js:256]: need to add env vars for that

File: server/src/routes/api/v3/wallets/index.js [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+ * @api {post} api/v3/wallets/foreign Create wallet contract for user on Ethereum
+ * @apiName CreateWalletForeign
+ * @apiGroup Wallet
+ * @apiDescription Creates wallet contract for the user on Ethereum
+ *
+ * @apiHeader {String} Authorization JWT Authorization in a format "Bearer {jwtToken}"
+ *
+ * @apiSuccess {Object} Started job data
+ */
+router.post('/foreign', auth.required, async (req, res, next) => {
```
Comment (leonprou) [inline server/src/routes/api/v3/wallets/index.js:263]: @LiorAgnin this job isn't supported yet actually

File: server/src/routes/api/v3/wallets/notify.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+  if (userWallet.networks.includes(network) || (userWallet.networks.pendingNetworks && userWallet.networks.pendingNetworks.includes(network))) {
+    return
+  }
+
+  const pendingNetworks = [...userWallet.pendingNetworks, network]
+  await UserWallet.findOneAndUpdate({ walletAddress }, { pendingNetworks })
+
+  console.log(`starting a createForeignWallet job for ${JSON.stringify({ walletAddress: userWallet.walletAddress, network })}`)
+  // const job = await agenda.now('createForeignWallet', { userWallet, network: config.get('network.foreign.name') })
```
Comment (leonprou) [inline server/src/routes/api/v3/wallets/notify.js:116]: I think we can stop creating foreign wallets on Ethereum for now, anyway the blocknative watch is not working. @LiorAgnin what you  think?

Comment (LiorAgnin) [reply server/src/routes/api/v3/wallets/notify.js:116]: @leonprou Yes, I agree.

**Review by LiorAgnin — DISMISSED** [review]
LGTM!

**Review by LiorAgnin — APPROVED** [review]

**Pull request merged**